### PR TITLE
Update address type leaderboard

### PR DESCRIPTION
**Pull request by thomasyu888**
make sure to only show submissions that have score > 0

### CODE REVIEW

**Comment by tschaffter** [comment]
@thomasyu888 Is this PR still required or you are filtering out scores that are NA? I submission may technically get a score equal to 0. It should still show up in the table.

**Comment by thomasyu888** [comment]
@tschaffter Yao had told me that you guys discussed that if annotations are missing address type, that the scores would be 0.  So I am filtering out all scores that are 0.

**Comment by tschaffter** [comment]
@yy6linda I don't think that we have agree with what Tom mentioned. That's what NA is for. Otherwise if we filter out submission with a 0 score, there is a chance that a submission DOES output address type but is just very bad at it.

**Comment by yy6linda** [comment]
Hi @tschaffter, what the evaluation script  is doing right now is for dateFormat and personType that we don't have in the goldstandard we return "NA". We have address type in the goldstandard, so addressType will always be evaluated. If participants don't provide addressType in their annotation, they will see a 0 instead of "NA"

**Comment by tschaffter** [comment]
> We have address type in the goldstandard, so addressType will always be evaluated. If participants don't provide addressType in their annotation, they will see a 0 instead of "NA".

That's not correct. We should only do the evaluation IF there is least one annotation with the address type in the gold standard AND IF there is at least one annotation with the address type in the prediction. The current solution leads to the issue I described above.

**Comment by yy6linda** [comment]
A participant can provide addressType annotation for some objects but not for all. For this case, it's hard for us to track if participants have provided any addresstype annotation at all. It makes sense to me that they get a score of 0. It's different from NA that we are not capable of conducting the evaluation due to the missingness of goldstandard.

**Comment by tschaffter** [comment]
> For this case, it's hard for us to track if participants have provided any addresstype annotation at all.

What do you mean by "hard"? Isn't it simply about iterating over a list of objects and check that 1) the property `addressType` of the object exists and 2) that the value of this property is valid?

**Comment by yy6linda** [comment]
I can work on changing it.

**Comment by yy6linda** [comment]
@tschaffter @thomasyu888 Should be fixed now by this [PR](https://github.com/nlpsandbox/nlpsandbox-client/pull/111)

File: wiki/608041-PhysicalAddressAnnotationLeaderboard.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ On"%2C submitterid as Submitter%2C dataset%5Fversion as "Dataset Version"%2C
 type%5FF1  as "F1 Type"%2C type%5Frecall as "Recall Type"%2C type%5Fprecision as
 "Precision Type"%2C dockerrepositoryname as Repository%2C dockerdigest as Digest
 from  syn23747126 where evaluationid %3D 9614658 and status %3D
-%27ACCEPTED%27&showquery=false}
+%27ACCEPTED%27 and type%5FF1 <> 0&showquery=false}
```
Comment (thomasyu888) [inline wiki/608041-PhysicalAddressAnnotationLeaderboard.md]:
```suggestion
%27ACCEPTED%27 and type%5FF1 is not null&showquery=false}
```

**Review by tschaffter — APPROVED** [review]
This PR looks good to me!

**Pull request merged**